batch-import/post_content.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# POST a single video to a Box for segmentation and analysis
+# This requires that
+# 1. You have a Camio Box running and you can access it over the network
```

Comment:
Just don't want people to think "over the network" as over the Internet and start opening ports when they don't have to do that. So perhaps just revise to say "access it on your local network or over the Internet"